### PR TITLE
Revises to specify image file types used for bootable usb drive on Wi…

### PR DESCRIPTION
**Pull request by mvincerx**
…ndows.

- Revises include statement to exclude table
- Removes unused 'releases' link

Signed-off-by: Michael Vincerra <michaelx.vincerra@intel.com>

### CODE REVIEW

File: source/clear-linux/get-started/bootable-usb/bootable-usb-windows.rst [hunk 1]
```diff
@@ -4,18 +4,18 @@ Create a bootable USB drive on Windows
 ######################################
 
 Follow these instructions to create a bootable |CLOSIA| USB drive.
-Use an **8GB** or larger USB drive.
+Use an **8GB** or larger USB drive.Download either a live image,
```
Comment (tpleavitt) [inline source/clear-linux/get-started/bootable-usb/bootable-usb-windows.rst]: Looks like you need a space between the period and Download

File: source/clear-linux/get-started/bootable-usb/bootable-usb-windows.rst [hunk 2]
```diff
@@ -4,18 +4,18 @@ Create a bootable USB drive on Windows
...
 
 * :ref:`bootable-usb-mac`
 * :ref:`bootable-usb-linux`
 
 .. include:: ../../guides/maintenance/image-types.rst
-   :start-after: image-types-content:
+   :start-after: for different platforms and environments. 
+   :end-before: Table 1 lists the currently available images.
 
 .. include:: ../../guides/maintenance/download-verify-uncompress-windows.rst
```
Comment (tpleavitt) [inline source/clear-linux/get-started/bootable-usb/bootable-usb-windows.rst]: This didn't show up in the rendered file

Comment (mvincerx) [reply source/clear-linux/get-started/bootable-usb/bootable-usb-windows.rst]: @tpleavitt The ReST includes don't show up in the rendered file on GitHub. 

Please compare the linux version that I pushed in PR 145. While that 'include' did not show up in rendered file on GitHub, it does however, show up on the build.  
Compare: 
https://github.com/mvincerx/clear-linux-documentation/blob/e33e17c584ae706fad2419c6b4b32d8d36f7fee4/source/clear-linux/get-started/bootable-usb/bootable-usb-linux.rst

With: 
https://clearlinux.org/documentation/clear-linux/get-started/bootable-usb/bootable-usb-linux

Comment (mvincerx) [reply source/clear-linux/get-started/bootable-usb/bootable-usb-windows.rst]: @tpleavitt Please see second commit below.

**Pull request merged**